kcidb/monitor/spool/__init__.py

```diff
@@ -97,6 +97,11 @@ def post(self, notification, timestamp=None):
             timestamp:      An "aware" datetime.datetime object specifying the
                             notification creation time, or None to use
                             datetime.datetime.now(datetime.timezone.utc).
+            due:            An "aware" datetime.datetime object specifying the
+                            time notification should be sent out,
+                            or None to use
+                            datetime.datetime.now(datetime.timezone.utc).
```

Comment:
Let's make the default be the value of the `timestamp` argument, thus making `due` match the creation time, making the notification due for sending immediately.